src/main/java/hunternif/mc/atlas/item/ItemEmptyAtlas.java

```diff
@@ -14,7 +16,10 @@
 import net.minecraftforge.common.ForgeHooks;
 
 public class ItemEmptyAtlas extends Item {
+
 	public ItemEmptyAtlas() {
+	    setUnlocalizedName("emptyAntiqueAtlas");

```

Comment:
The entire project is littered with tabs and spaces. What needs to happen is a project-wide formatting.